vividus-extension-web-app/src/main/java/org/vividus/ui/web/action/IBaseCookieManager.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.ui.web.action;
+
+public interface IBaseCookieManager<T, S>
```

Comment:
```suggestion
public interface CookieManager<T, S>
```
(feel free to rename current classes to `PlaywrightCookieManager` and `WebDriverCookieManager`)

Reply:
updated, thanks